src/components/dynamic-lists/dynamic-list.component.ts

```diff
@@ -88,11 +90,36 @@ export class BeagleListViewComponent
     })
   }
 
+  getColumnsQuantityStyle() {
+    return `repeat(${this.numColumns}, auto)`
+  }
+
+  getClassForType() {
+    return this.type === 'GRID' ? 'beagle-grid-view' :
```

Comment:
Use the suggestion of creating a String Enum for Type and Direction.

```suggestion
    return this.type === ListType.Grid ? 'beagle-grid-view' :
```